Makefile

```diff
@@ -16,85 +69,138 @@ endif
 # Setting SHELL to bash allows bash commands to be executed by recipes.
 # This is a requirement for 'setup-envtest.sh' in the test target.
 # Options are set to exit when a recipe line exits non-zero or a piped command fails.
-SHELL = /usr/bin/env bash -o pipefail
+# SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
-# Allow overriding release-manifest generation destination directory
-RELEASE_DIR ?= out
-
 # Quiet Ginkgo for now.
 # The warnings are in regards to a future release.
 export ACK_GINKGO_DEPRECATIONS := 1.16.5
 export ACK_GINKGO_RC=true
 
-export PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-export PATH := $(PROJECT_DIR)/bin:$(PATH)
+export PATH := $(TOOLS_BIN_DIR):$(PATH)
 
 all: build
 
-##@ General
+## --------------------------------------
+## Binaries
+## --------------------------------------
 
-# The help target prints out all targets with their descriptions organized
-# beneath their categories. The categories are represented by '##@' and the
-# target descriptions by '##'. The awk commands is responsible for reading the
-# entire set of makefiles included in this invocation, looking for lines of the
-# file as xyz: ## something, and then pretty-format the target and help. Then,
-# if there's a line with ##@ something, that gets pretty-printed as a category.
-# More info on the usage of ANSI control characters for terminal formatting:
-# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
-# More info on the awk command:
-# http://linuxcommand.org/lc3_adv_awk.php
+.PHONY: binaries
+binaries: $(CONTROLLER_GEN KUSTOMIZE) $(GOLANGCI_LINT) $(STATIC_CHECK) $(GINKGO_V1) $(GINKGO_V2) $(MOCKGEN) $(KUSTOMIZE) managers # Builds and installs all binaries
 
-help: ## Display this help.
```

Comment:
Why are removing the help command and it's output?

Reply:
It is there in the common.mk
That way any file which needs a help can import it instead of adding it's own

```
[10:34:02] ~/lab/capi ➦ standardize-makefile $ make help

Usage:
  make <target>
  manager-cloudstack-infrastructure  Build manager binary.
  fmt              Run go fmt on the whole project.
  vet              Run go vet on the whole project.
  lint             Run linting for the project.
  modules          Runs go mod to ensure proper vendoring.
  generate-mocks   Generate mocks needed for testing. Primarily mocks of the cloud package.
  generate-deepcopy  Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
  generate-manifests  Generates crd, webhook, rbac, and other configuration manifests from kubebuilder instructions in go comments.
  build            Build manager binary.
  build-for-docker  Build manager binary for docker image building.
  run              Run a controller from your host.
  deploy           Deploy controller to the K8s cluster specified in ~/.kube/config.
  undeploy         Undeploy controller from the K8s cluster specified in ~/.kube/config.
  docker-build     Build docker image containing the controller manager.
  docker-push      Push docker image with the manager.
  tilt-up          Setup and run tilt for development.
  kind-cluster     Create a kind cluster with a local Docker repository.
  cluster-api      Clone cluster-api repository for tilt use.
  test             Run tests. At the moment this is only unit tests.
  clean            Cleans up everything.
  release-manifests  Create kustomized release manifest in $RELEASE_DIR (defaults to out).
  release-staging  Builds and push container images and manifests to the staging bucket.
  release-templates  Generate release templates
  upload-staging-artifacts  Upload release artifacts to the staging bucket
  help             Display this help

```

Reply:
Thank you for cleaning up the help menu and targets/descriptions.